### DISCUSSION TITLE
Max value of color scale

### DISCUSSION THREAD

**Discussion by idoidoido** [sgratzl/chartjs-chart-geo#81] [General]
Hi!

How can I get current max value of color scale?
In barchart, I can get current max value like this.

myChart.scales['y-axis-0'].max

And, I can set max value using this.

myChart.options.geo.colorScale.ticks.max

But, this value is empty in default.
I would like to know current max value of color scale which is automatically adjusted using max value of dataset.

**Comment by stockiNail** [comment]
@idoidoido  as far as  I know

> In barchart, I can get current max value like this.
> 
> myChart.scales['y-axis-0'].max

This is the same also for color scale, you can get the max value of axis.

```javascript
myChart.scales.color.max
```

See console of this codepen: https://codepen.io/stockinail/pen/WNpqEyd

**Comment by idoidoido** [comment]
Thank you for your reply.

I cannot find 'color' in myChart.scales.
I am using chart.js v2.9.1 and chartjs-chart-geo@2.1.0.
Is this by version difference?

**Comment by CesarDiax** [comment]
Que tal me podran ayudar con que con un mapa que sea lo de mexico. scale: {projection: 'Mexico' }
